content/doc/book/managing/security.adoc

```diff
@@ -429,7 +429,7 @@ level of access to all configured projects.
 An administrator can disable Agent/Master Access Control in the web UI by
 un-checking the box on the *Configure Global Security* page. Alternatively an
 administrator may create a file in `JENKINS_HOME/secrets` named
-`slave-to-master-security-kill-switch` with the contents of `true` and restart
+`slave-to-controller-security-kill-switch` with the contents of `true` and restart
```

Comment:
This introduced an obvious mistake in the documentation.

Could we please actually review changes @MarkEWaite ?